scripts/src/java/org/oppia/android/scripts/common/ScriptExemptions.kt

```diff
@@ -0,0 +1,621 @@
+    "HintsAndSolutionExplorationManagerFragmentPresenter.kt",
+    "ExplorationViewModel.kt",
+    "ExplorationFragment.kt",
+    "ExplorationManagerFragmentPresenter.kt",
+    "DefaultFontSizeStateListener.kt",
+    "ExplorationActivityPresenter.kt",
+    "ExplorationFragmentPresenter.kt",
+    "WalkthroughPages.kt",
+    "WalkthroughWelcomeFragmentPresenter.kt",
+    "WalkthroughWelcomeViewModel.kt",
+    "WalkthroughFragmentChangeListener.kt",
+    "WalkthroughViewModel.kt",
+    "WalkthroughActivityPresenter.kt",
+    "WalkthroughActivityListener.kt",
+    "WalkthroughPageChanger.kt",
+    "WalkthroughTopicViewModel.kt",
+    "WalkthroughTopicSummaryViewModel.kt",
+    "WalkthroughTopicHeaderViewModel.kt",
+    "WalkthroughTopicListFragmentPresenter.kt",
+    "WalkthroughTopicItemViewModel.kt",
+    "WalkthroughFinalViewModel.kt",
+    "WalkthroughFinalFragmentPresenter.kt",
+    "WalkthroughFinalListener.kt",
+    "WalkthroughEndPageChanger.kt",
+    "ProfileEditDialogInterface.kt",
+    "ProfileEditFragment.kt",
+    "ProfileEditViewModel.kt",
+    "ProfileResetPinActivityPresenter.kt",
+    "ProfileEditActivityPresenter.kt",
+    "ProfileListFragmentPresenter.kt",
+    "ProfileListActivityPresenter.kt",
+    "ProfileListViewModel.kt",
+    "ProfileEditDeletionDialogFragment.kt",
+    "ProfileEditFragmentPresenter.kt",
+    "ProfileRenameActivityPresenter.kt",
+    "ProfileResetPinViewModel.kt",
+    "ProfileRenameViewModel.kt",
+    "RecyclerViewMatcher.kt",
+    "EspressoTestsMatchers.kt",
+    "ProgressMatcher.kt",
+    "OrientationChangeAction.kt",
+    "FontSizeMatcher.kt",
+    "DragViewAction.kt",
+    "DrawableMatcher.kt",
+    "TabMatcher.kt",
+    "ClickActions.kt",
+    "Constants.kt",
+    "OppiaRetrofit.kt",
+    "NetworkSettings.kt",
+    "GaeState.kt",
+    "GaeExplorationContainer.kt",
+    "GaeSolution.kt",
+    "GaeUserSuppliedFeedback.kt",
+    "GaeStorySummary.kt",
+    "GaeStory.kt",
+    "GaeStoryNode.kt",
+    "GaeWrittenTranslation.kt",
+    "GaeExpSummary.kt",
+    "GaeSubtitledHtml.kt",
+    "GaeInteractionInstance.kt",
+    "GaeStateClassifier.kt",
+    "GaeFeedbackReportingSystemContext.kt",
+    "GaeRuleSpec.kt",
+    "GaeQuestionPlayer.kt",
+    "GaeRecordedVoiceovers.kt",
+    "GaeTopic.kt",
+    "GaeConceptCard.kt",
+    "GaeQuestion.kt",
+    "GaeVoiceover.kt",
+    "GaeTopicSummary.kt",
+    "GaeParamChange.kt",
+    "GaeClassroom.kt",
+    "GaeAnswerGroup.kt",
+    "GaeFeedbackReport.kt",
+    "GaeExploration.kt",
+    "GaeSubtopic.kt",
+    "GaeParamSpec.kt",
+    "GaeSkillContents.kt",
+    "GaeSubtopicPageContents.kt",
+    "GaeWrittenTranslations.kt",
+    "GaeFeedbackReportingAppContext.kt",
+    "GaeFeedbackReportingDeviceContext.kt",
+    "GaeCustomizationArgs.kt",
+    "GaeOutcome.kt",
+    "GaeFeedbackReportingEntryPoint.kt",
+    "GaeHint.kt",
+    "GaeSubtopicSummary.kt",
+    "NetworkApiKey.kt",
+    "NetworkModule.kt",
+    "ScriptExemptions.kt",
+    "ScriptResultConstants.kt",
```

Comment:
@BenHenning Added these two files because they only contain constants or a list of constants. So, I think it is fine to exempt for the check.

Reply:
The constants one came to mind to me as well--we may need to find a way to properly exempt files which do not ever need to have tests. Maybe if the file doesn't have an interface, annotation, function, or class?

Or, maybe we should just move the constants to their corresponding files instead of having a shared constants file. Does that file actually provide any specific benefits?

Reply:
Will remove the constants file as it is not of any benefit.
Will do this after merging this PR with #3340.

Reply:
Update: This is now removed.